src/policy/fees.h

```diff
@@ -185,20 +186,30 @@ class CBlockPolicyEstimator
      */
     static constexpr double FEE_SPACING = 1.05;
 
+    struct TxStatsInfo
+    {
+        unsigned int blockHeight;
+        CAmount m_fee_per_k;
+        TxStatsInfo() : blockHeight(0), m_fee_per_k(0) {}
+    };
+
+    /** Remove a transaction from the mempool tracking stats*/
+    void removeTx(std::map<uint256, TxStatsInfo>::iterator pos, bool inBlock);
```

Comment:
Happy to rename it, but honestly I dont remember what the different responsibilities are. Its been so long since I wrote this I dont know how it works anymore. Any suggestions?

Reply:
Actually, it's fine leaving this one as is because in the later commit (Process fee estimates in the background via MempoolInterface), the other `removeTx` gets renamed to `removeTxNotInBlock`.

Not really necessary, but maybe rename the other function to `removeTxNotInBlock` in this commit instead of a later commit to avoid the confusion here.